api/v1/controllers/subjects.js

```diff
@@ -33,6 +34,9 @@ const apiErrors = require('../apiErrors');
 const redisSubjectModel = require('../../../cache/models/subject');
 const sampleStoreFeature =
                   require('../../../cache/sampleStore').constants.featureName;
+const jobType = require('../../../jobQueue/setup').jobType;
+const jobWrapper = require('../../../jobQueue/jobWrapper');
+const WORKER_TTL = 30000;
 const ZERO = 0;
```

Comment:
30 second timeout is universal in heroku, applies to everything.

> The countdown for this 30 second timeout begins after the entire request (all request headers and, if applicable, the request body) has been sent from the router to the dyno. The request must then be processed in the dyno by your application, and a response delivered back to the router, within 30 seconds to avoid the timeout.